main/vlan/mvlan.pre-up

```diff
@@ -22,9 +27,7 @@ if ! ip link show "$RAW_DEVICE" >/dev/null; then
 	exit 1
 fi
 
-ip link add link "$RAW_DEVICE" name "$IFACE" type macvlan
-
-if ! ip link show "$IFACE" >/dev/null; then
-	echo "Failed to create vlan device $IFACE on device $RAW_DEVICE with tag $VLAN_ID"
-	exit 1
+trap "device_creation_error" ERR
+if ! ip link show "$IFACE" &>/dev/null; then
```

Comment:
`&>` is a bashism. Please replace it with `>/dev/null 2>&1`.

Reply:
Fixed, thanks